packages/mobile/src/backup/BackupIntroduction.tsx

```diff
@@ -54,7 +54,7 @@ class BackupIntroduction extends React.Component<Props> {
   }
 
   onPressBackup = () => {
-    CeloAnalytics.track(CustomEventNames.set_backup_phrase)
```

Comment:
is this deprecated or still being used elsewhere? if deprecated, can we delete from the constants file?